transforms/code/header_cleanser/python/src/header_cleanser_transform.py

```diff
@@ -0,0 +1,221 @@
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+################################################################################
+
+import os
+import tempfile
+from argparse import ArgumentParser, Namespace
+
+import pyarrow as pa
+from data_processing.runtime.pure_python.runtime_configuration import (
+    PythonTransformRuntimeConfiguration,
+)
+from data_processing.transform import AbstractTableTransform, TransformConfiguration
+from data_processing.utils import get_logger,CLIArgumentProvider
+from scancode import api
+
+logger = get_logger(__name__)
+
+short_name = "header_cleanser"
+cli_prefix = short_name + "_"
+COLUMN_KEY = "contents_column_name"
+LICENSE_KEY = "license"
+COPYRIGHT_KEY = "copyright"
+
+column_cli_params = f'{cli_prefix}{COLUMN_KEY}'
+license_cli_params = f'{cli_prefix}{LICENSE_KEY}'
+copyright_cli_params = f'{cli_prefix}{COPYRIGHT_KEY}'
+
+DEFAULT_COLUMN = "contents"
+DEFAULT_LICENSE = "true"
+DEFAULT_COPYRIGHT = "true"
+
```

Comment:
These should be boolean

Reply:
boolean type gives error while running make-test. That's why I used str format.
error :
ERROR collecting test/test_license_copyright_removal.py ________________________
test_license_copyright_removal.py:70: in get_test_transform_fixtures
self.create_license_copyright_removal_test_fixture(
test_license_copyright_removal.py:52: in create_license_copyright_removal_test_fixture
config = get_transform_config(ftc, cli)
../../../../../data-processing-lib/python/src/data_processing/transform/transform_configuration.py:100: in get_transform_config
args = parser.parse_args(argv)
/usr/lib/python3.11/argparse.py:1886: in parse_args
args, argv = self.parse_known_args(args, namespace)
/usr/lib/python3.11/argparse.py:1919: in parse_known_args
namespace, args = self._parse_known_args(args, namespace)
/usr/lib/python3.11/argparse.py:1962: in _parse_known_args
option_tuple = self._parse_optional(arg_string)
/usr/lib/python3.11/argparse.py:2261: in _parse_optional
if not arg_string[0] in self.prefix_chars:
E TypeError: 'bool' object is not subscriptable

Reply:
Yes, take a look at https://github.com/IBM/data-prep-kit/blob/dev/transforms/code/code2parquet/python/src/code2parquet_transform.py#L184

Reply:
Updated. please check